docs/architecture.md

```diff
@@ -0,0 +1,353 @@
+# Architecture
+
+```mermaid
+flowchart LR
+
+SDK[cow-sdk]
+OrderBookApi
+SubgraphApi
+MetadataApi
+
+SDK --> OrderBookApi
+SDK --> SubgraphApi
+SDK --> MetadataApi
+```
+
+The SDK has 3 main APIs
+
+- **Order Book API**: Allows to get the open orders, historic orders, post new orders, etc.
+- **Subgraph API**: Provides access to On-chain data indexed by The Graph
+- **Metadata API**: Allows to encode/decode meta-data to be attached in orders
```

Comment:
This has actually been removed/moved to @cowprotocol/app-data

Reply:
Yep, I know :) 

I consider deleting, but I think we should move the docs to that project too. And I don't want to step too much in the Sasha's part. I just wanted to try the PLANTUML and make a proposal with Marmade UML. 

Let's merge and then we can continue to move things around and do the finer adjustments to it